.github/workflows/test.yml

```diff
@@ -162,3 +162,24 @@ jobs:
           mv /tmp/.buildx-cache-new /tmp/.buildx-cache
       - name: Image digest
         run: echo ${{ steps.docker_build.outputs.digest }}
+
+  autodeploy:
+    runs-on: ubuntu-latest
+    needs: [docker-deploy]
```

Comment:
`docker-deploy` is not a declared job or did I miss something?

Reply:
it's the name of the step before this, it has the same name as the rest of the projects.

Reply:
The name in this project is `docker-publish`

Reply:
True, my fault, I corrected it. Thanks